Tests/FunctionalTests/AEPEdgeFunctionalTests.swift

```diff
@@ -22,7 +22,7 @@ import XCTest
 
 /// End-to-end testing for the AEPEdge public APIs
 class AEPEdgeFunctionalTests: TestBase, AnyCodableAsserts {
-    private let TIMEOUT_SEC: TimeInterval = 2
+    private let TIMEOUT_SEC: TimeInterval = 10
     private let LONGER_TIMEOUT_SEC: TimeInterval = 10
```

Comment:
We have LONGER_TIMEOUT_SEC which is 10 seconds. We can move this into common constants file (FuntionalTestConstants.swift) if this applies to all the integration tests, and if let's use only one time and update it to 10 sec if that makes our tests more stable.